BackofficeBundle/Tests/Form/Type/DatePickerTypeTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\Tests\Form\Type;
+
+use OpenOrchestra\BackofficeBundle\Form\Type\DatePickerType;
+
+/**
+ * Class DatePickerTypeTest
+ */
+class DatePickerTypeTest extends \PHPUnit_Framework_TestCase
```

Comment:
You should emphasize your test to test the finish view method in his whole scope